src/components/booksContainer/BooksContainer.js

```diff
@@ -1,18 +1,28 @@
-import { useSelector } from 'react-redux';
+/* eslint-disable arrow-body-style */
+/* eslint-disable no-unused-vars */
+import { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { getBooksFromApi } from '../../redux/books/books';
 import Book from '../book/Book';
 
 const BooksContainer = () => {
   const books = useSelector((state) => state.booksReducer);
+  const dispatch = useDispatch();
 
-  const createBooks = (booksCollection) => booksCollection.map((book) => (
-    <Book
-      key={Math.random()}
-      id={book.id}
-      author={book.author}
-      title={book.title}
-      description={book.description}
-    />
-  ));
+  useEffect(() => {
+    dispatch(getBooksFromApi());
+  }, []);
+
+  const createBooks = (booksCollection) => booksCollection.map((book) => {
+    return (
+      <Book
+        key={book.item_id}
+        id={book.item_id}
+        category={book.category}
+        title={book.title}
```

Comment:
* Kindly fix this error in the console i get them when i try to add new books 
![image](https://user-images.githubusercontent.com/52497246/130699929-d97ec210-47dd-48a4-9584-dd252818184e.png)

Reply:
Done!.. I think... changed the id generator for uuid.